lib/ansible/modules/cloud/amazon/_ec2_vpc.py

```diff
@@ -25,6 +25,10 @@
 description:
     - Create or terminates AWS virtual private clouds.  This module has a dependency on python-boto.
 version_added: "1.4"
+deprecated: >-
+  Deprecated in 2.3. Use M(epc_vpc_net) along with supporting modules including
```

Comment:
@willthames It looks like a typo made it through here. Shouldn't that be `ec2_vpc_net` not `epc_vpc_net`?